src/encdatavault_fmt_plug.c

```diff
@@ -108,14 +131,16 @@ static size_t cracked_size;
 
 static char (*saved_key)[PLAINTEXT_LENGTH + 1];
 
-static struct custom_salt {
+static struct enc_salt {
 	unsigned int version;
 	unsigned int algo_id;
-	unsigned char salt[ENC_SALT_SIZE];
 	unsigned char iv[ENC_IV_SIZE];
+	unsigned int salt_length;
+	unsigned char salt[PBKDF2_32_MAX_SALT_SIZE];
+	unsigned int iterations;
 	unsigned char encrypted_data[ENC_BLOCK_SIZE];
 	unsigned char keychain[ENC_KEYCHAIN_SIZE];
-} *cur_salt;
+} *enc_cur_salt;
```

Comment:
I don't see a reason to rename this struct and pointer. IIRC, we don't do that in other formats, except where this gets moved into an `.h` file for sharing between CPU and OpenCL formats.